test/worlds/diff_drive.sdf

```diff
@@ -2,9 +2,9 @@
 <sdf version="1.6">
   <world name="diff_drive">
 
-    <physics name="1ms" type="ode">
+    <physics name="1ms" type="ignored">
```

Comment:
Why make this change?

Reply:
I was just matching most of the other files.

A while back we changed all referenced to `ode` to `ignored` (https://github.com/ignitionrobotics/ign-gazebo/commit/23b1f71a2441b5bb80927bf6413d08d63b4a6254) so users don't have the wrong impression that we're using ODE, or that the field is being used. We should eventually let the user choose the physics engine from that argument, or completely deprecate it.